File: src/test/java/com/james/PersonTest.java
package com.james;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class PersonTest {
    @Test
    public void bmiTest() {
        Person person = new Person(66.5f, 1.7f);
        Assertions.assertEquals(66.5f / (1.7f * 1.7f), person.bmi());
    }
}

File: src/test/kotlin/com/james/PersonTest.kt
package com.james.kotlin

import com.james.Human
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class PersonTest {
    @Test
    fun humanBmiTest() {
        val human = Human("james", 66.5f, 1.7f)
        Assertions.assertEquals(66.5f / (1.7f * 1.7f), human.bmi())
    }

}